encode-decode/JwtDecode.js

```diff
@@ -0,0 +1,42 @@
+// JWT Decode by 0mgfriday
+var Base64 = Java.type("java.util.Base64")
+var String = Java.type("java.lang.String")
+
+/**
+ * Decode JWT into JSON
```

Comment:
It does not decode into JSON, "into a text representation" would be more accurate.

Reply:
Updated